### PR TITLE
Recovery date corrected and new mission yml added

### PR DESCRIPTION
**Pull request by akmazur**

### CODE REVIEW

File: mission_yaml/SEA55_M33.yml [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+  #   serial: '2043'
+  #   long_name: SeaBird OCR504
+  #   make_model: SeaBird OCR504
+  #   factory_calibrated: "Yes"
+  #   calibration_date: "2020-02-25"
+
+# map between glider variables and netcdf variables.  This shouldn't
+# change too much.
+netcdf_variables:
+  keep_variables: ['temperature', 'oxygen_concentration', 'chlorophyll', 'downwelling_PAR']
```
Comment (Maribaken) [inline mission_yaml/SEA55_M33.yml:90]: This should (probably) be removed

Comment (Maribaken) [reply mission_yaml/SEA55_M33.yml:90]: downwelling_PAR that is

Comment (akmazur) [reply mission_yaml/SEA55_M33.yml:90]: right, didn't see it here

**Review by Maribaken — APPROVED** [review]
I think this looks good, let's give it a try

**Pull request merged**